### ISSUE TITLE
transition attribute/property in css set to 0s or none

### ISSUE THREAD

**Issue by Joystickplays** [michalsnik/aos#732]
## This is:

- Bug

## Specifications

  - AOS version: 2.3.4
  - OS: Windows 10
  - Browser: Chrome
  - 
## Expected Behavior
`text` gets animated on scroll, and when on hover it should increase the padding with a transition.

## Actual Behavior
`text` gets animated on scroll, and when on hover it increased the padding instantly.

## Steps to Reproduce the Problem
1. Create a `<p>` tag with class `text`
2. Add the property `transition` to the `text` class with a value of `0.3s` 
3. Make the class `text` increase padding on hover
4. Add `data-aos="fade-up` to the `<p>` tag

## Detailed Description
Basically I have a `<p>` element (classed in `text`) that I want to animate `fade-up`. In my CSS, I make `text` set the padding to 16px with 0.3s of transition.
`text {
  transition: 0.3s;
}

.text:hover {
  padding: 16px;
}
But when I added `data-aos="fade-up"`, the transition is gone, like it's been set to 0s.
